libs/brig-types/src/Brig/Types/Team/Invitation.hs

```diff
@@ -49,7 +49,8 @@ instance ToJSON InvitationRequest where
 instance FromJSON Invitation where
     parseJSON = withObject "invitation" $ \o ->
         Invitation <$> o .: "team"
-                   <*> o .:? "role"
+                       -- clients don't can leave the default role choice to us:
```

Comment:
```suggestion
                       -- clients, when leaving "role" empty, can leave the default role choice to us
```

Reply:
What's the swagger for `Invitation`? Does that indicate role is optional or required?